common.sh

```diff
@@ -418,7 +418,6 @@ parse_txt_junit_xml()
 {
     exit_code=$?
     set +x
-    pushd ${output_dir}
```

Comment:
The push/popds should work as a stack regardless of the push/popds inside the functions we call. It is possible we are missing a popd somewhere, might be better to find and fix that. But this is fine as-is too.